index.html

```diff
@@ -257,8 +257,47 @@ <h4 class="ab-titles skills"> Skills</h4>
         </div>
       </div>
     </section>
+    <section class="form-container flex-column">
+      <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <div class="form flex-column">
+        <form class="flex-column " action="https://formspree.io/f/mayzgdra" method="POST" class="form-data">
+          <input type="text" id="name" placeholder="Full name" required maxlength="30">
+          <input type="email" id="email" placeholder="Email address" required>
+          <input type="text" id="message" placeholder="Write something..." required maxlength="500">
+          <center><button type="submit" class="button-resume">Get in touch</button></center>
+          <img src="/img/Rectangle 42.png" alt="" class="vector-footer">
+        </form>
```

Comment:
You did a great job integrating **Formspree** in your form as an `action` in your form. 🚀 However, your form submission does not work as expected. **Formspree** requires a `name` attribute in order to match the input field.
 
![image](https://github.com/Rebelzob/Portfolio/assets/117144028/708013cf-15af-4e80-abc0-4e53b8207ef8)

- [ ] Please refer [here](https://formspree.io/html/) to check the proper implementation of **Formspree** forms. 🎯